packages/react/src/components/Placeholder/models.ts

```diff
@@ -24,6 +24,7 @@ export interface AppComponentProps {
     [name: string]: string;
   };
   rendering: ComponentRendering;
+  disableSuspense?: boolean;
```

Comment:
Not sure I follow why our components should get a `disableSuspense` prop - how is this supposed to be used?

Reply:
We passed *all* props to components with old Placeholder implementation, but we filter them with AppPlaceholder implementation,  as it needs to be safe while not passing non-serializable props.
disableSuspense is serializable, but was cut from propagation somewhere along the line, and was not passed into child placeholders.